src/config/config.php

```diff
@@ -1,22 +1,45 @@
 <?php
-return array(
-	// which type of store to use.
-	// valid options: 'json', 'database'
+
+return [
```

Comment:
Was this intentional to convert from `array()` to `[]`, the rest of the code base is using `array()`